erizo/src/erizo/thread/Scheduler.cpp

```diff
@@ -0,0 +1,90 @@
+#include "thread/Scheduler.h"
+
+#include <assert.h>
+
+#include <boost/bind.hpp>
+#include <utility>
+
+
+Scheduler::Scheduler(int n_threads_servicing_queue)
+: n_threads_servicing_queue_(0), stop_requested_(false), stop_when_empty_(false) {
```

Comment:
I finally apply your comment to avoid potential race conditions while running unit tests.